main.go

```diff
@@ -8,7 +8,6 @@ import (
 	"runtime"
 
 	"github.com/harness/harness-docker-runner/cli"
-	_ "github.com/joho/godotenv/autoload"
```

Comment:
Don't we need this ?

Reply:
added back this _ "github.com/harness/godotenv/v3/autoload" instead